v2/assistantv1/assistant_v1_suite_test.go

```diff
@@ -17,9 +17,10 @@
 package assistantv1_test
 
 import (
+	"testing"
+
 	. "github.com/onsi/ginkgo"
 	. "github.com/onsi/gomega"
-	"testing"
 )
```

Comment:
is the only change on this file just adding a new line?

Reply:
I see this on multiple suite_test files, so I'm just going to mention it on this comment

Reply:
Yes the linter decided that standard library imports should be above package imports